lib/iris/analysis/_interpolation.py

```diff
@@ -554,9 +571,20 @@ def _points(self, sample_points, data, data_dims=None):
             # Transpose data in preparation for interpolation.
             data = np.transpose(data, interp_order)
 
-        # Interpolate and reshape the data ...
-        result = self._interpolate(data, interp_points)
-        result = result.reshape(interp_shape)
+        # Interpolate the data, merging the chunks in the interpolated
+        # dimensions.
+        dims_merge_chunks = [dmap[d] for d in di]
+        result = map_complete_blocks(
```

Comment:
TODO for @trexfeathers: confirm that this use of `map_complete_blocks()` does not suffer the memory leaks described in #5767.

Reply:
Have confirmed 👍